src/customer-service/Actions/CreateEmployeeAction.cs

```diff
@@ -0,0 +1,70 @@
+﻿using customer_service;
+using customer_service.Models;
+using System;
+
+namespace customer_service.Actions
+{
+
+    /**
+     * Class: CreateEmployeeAction
+     * Purpose: Prompts the user for information about a new employee and saves to DB
+     * Author: Matt Hamil
+     * Methods:
+     *     static void ReadInput() - Prompts for employee info and saves to DB
+     */
+    public class CreateEmployeeAction
+    {
+        /**
+         * Purpose: Reads user input for new employee information
+         * Return:
+         *     void
+         */
+        public static void ReadInput()
+        {
+            Employee newEmployee = new Employee();
+
+            while (newEmployee.FirstName == null || newEmployee.FirstName.Length <= 0)
+            {
+                Console.WriteLine("Enter employee first name: ");
+                newEmployee.FirstName = Console.ReadLine();
+            }
+
+            while (newEmployee.LastName == null || newEmployee.LastName.Length <= 0)
+            {
+                Console.WriteLine("Enter customer last name: ");
+                newEmployee.LastName = Console.ReadLine();
+            }
+
```

Comment:
Maybe overkill, but might be worth checking to see whether another Employee exists in the database with the same First and Last Name before we move on to Department